ui/object_names.h

```diff
@@ -0,0 +1,18 @@
+/*
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#define COMMAND_BUFFER_SEARCH_BAR_NAME "command_buffer_search_bar"
```

Comment:
it is generally no recommended to define macro in header in Google C++ Style Guide. You could probably change them into `const string kCommandBufferSearchBarName = "command_buffer_search_bar"`